### DISCUSSION TITLE
Tree view

### DISCUSSION THREAD

**Discussion by gabri83** [community/community#141692] [Mobile] [answered]
Any reason why there is no tree view for the mobile and desktop? It is super useful

**Comment by stevepopovich** [comment, marked as answer]
Hi @gabri83 are you referring to a tree view for commits?

**Reply by gabri83** [reply]
Yes, pretty much like the one you have on sourcetree or VSCode

**Reply by stevepopovich** [reply]
I filed this as a feature request, thank you!

**Reply by gabri83** [reply]
Thanks, it would be awesome to have!
